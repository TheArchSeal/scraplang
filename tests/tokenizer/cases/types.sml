0
'a'
""
a
(
)
[
]
{
}
+
++
-
--
*
/
%
|
||
&
&&
^
~
!
?
=
==
!=
<
<<
<=
>
>>
>=
+=
-=
*=
/=
%=
|=
&=
^=
<<=
>>=
->
=>
.
,
:
;
var
const
fn
wire
part
primitive
struct
enum
if
else
switch
case
default
while
do
for
return
break
continue
type
void
bool
i8
i16
i32
i64
u8
u16
u32
u64
