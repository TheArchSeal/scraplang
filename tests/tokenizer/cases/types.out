(2):1:1 0 0
(3):2:1 'a' a
(4):3:1 "" 
(5):4:1 a a
(6):5:1 (
(7):6:1 )
(8):7:1 [
(9):8:1 ]
(10):9:1 {
(11):10:1 }
(12):11:1 +
(13):12:1 ++
(14):13:1 -
(15):14:1 --
(16):15:1 *
(17):16:1 /
(18):17:1 %
(19):18:1 |
(20):19:1 ||
(21):20:1 &
(22):21:1 &&
(23):22:1 ^
(24):23:1 ~
(25):24:1 !
(26):25:1 ?
(27):26:1 =
(28):27:1 ==
(29):28:1 !=
(30):29:1 <
(31):30:1 <<
(32):31:1 <=
(33):32:1 >
(34):33:1 >>
(35):34:1 >=
(36):35:1 +=
(37):36:1 -=
(38):37:1 *=
(39):38:1 /=
(40):39:1 %=
(41):40:1 |=
(42):41:1 &=
(43):42:1 ^=
(44):43:1 <<=
(45):44:1 >>=
(46):45:1 ->
(47):46:1 =>
(48):47:1 .
(49):48:1 ,
(50):49:1 :
(51):50:1 ;
(52):51:1 var
(53):52:1 const
(54):53:1 fn
(55):54:1 wire
(56):55:1 part
(57):56:1 primitive
(58):57:1 struct
(59):58:1 enum
(60):59:1 if
(61):60:1 else
(62):61:1 switch
(63):62:1 case
(64):63:1 default
(65):64:1 while
(66):65:1 do
(67):66:1 for
(68):67:1 return
(69):68:1 break
(70):69:1 continue
(71):70:1 type
(72):71:1 void
(73):72:1 bool
(74):73:1 i8
(75):74:1 i16
(76):75:1 i32
(77):76:1 i64
(78):77:1 u8
(79):78:1 u16
(80):79:1 u32
(81):80:1 u64
